frontend/src/ClassDetail.svelte

```diff
@@ -35,6 +35,18 @@
           .reduce((acc, val) => acc + val, 0).toFixed(2);
   }
 
+  function taskPoints(clazz, k) {
```

Comment:
Please fix the formatting of this function.

Reply:
Fixed in the next commit called "Updated totalPointsPerAssignedTask".